packages/ruleset-bundler/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@stoplight/spectral-ruleset-bundler",
+  "version": "1.0.0",
+  "homepage": "https://github.com/stoplightio/spectral",
+  "bugs": "https://github.com/stoplightio/spectral/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "engines": {
+    "node": ">=12.20"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "default": "./dist/index.js"
+    },
+    "./presets/*": {
+      "default": "./dist/presets/*.js"
+    },
+    "./plugins/*": {
+      "default": "./dist/plugins/*.js"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/spectral.git"
+  },
+  "dependencies": {
+    "@stoplight/path": "1.3.2",
+    "@stoplight/spectral-core": ">=1",
```

Comment:
Should we at least peg it to a major? Or is there some reason it's OK in this case to use such a wide version range?

Reply:
We don't consume the API of these packages, so it doesn't matter.
The only package we do rely on is `spectral-runtime` and this one is indeed pegged.
These packages are only here for the sake of https://github.com/stoplightio/spectral/pull/1819/files#diff-8938b40bb5a35f8551f5d19ba496c77a0e75c44e23c3992d3a8c869ee51b5d1aR1.
Technically, I could make them peerDependencies or something, but I'm not a huge fan of peerDependencies.